src/main/java/org/mockito/internal/creation/MockSettingsImpl.java

```diff
@@ -95,8 +98,9 @@ public MockSettings defaultAnswer(Answer defaultAnswer) {
         return this;
     }
 
-    public MockSettings useConstructor() {
+    public MockSettings useConstructor(Object... constructorArgs) {
```

Comment:
@ChristianSchwarz - good point, will do. Thanks!

@szczepiq:
1. Good idea, will do. Thanks!
2. Before writing tests, I think we need to decide what the correct behavior would be. IMHO, this should just throw an exception, the same way java would refuse to compile `new MyClass(null)` if you had both a `MyClass(Object)` and a `MyClass(String)` constructor. What do you think?